app/templates/application.hbs

```diff
@@ -4,7 +4,6 @@
         {{#link-to 'ladder'}}Ladder{{/link-to}}
         {{#link-to 'rules'}}Rules{{/link-to}}
         {{#link-to 'history'}}History{{/link-to}}
-        {{#link-to 'admin'}}Admin{{/link-to}}
```

Comment:
So that when logged in as admin you can switch between the pages easier. Just around the Admin link.